src/main.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const defaults = require('lodash.defaults');
+const riot = require('riot');
+require('./templates/offer-renderer.tag');
+require('./templates/permission-renderer.tag');
+require('./templates/duty-renderer.tag');
+require('./templates/constraint-renderer.tag');
+require('./templates/prohibition-renderer.tag');
+const OfferParser = require('./offer-parser');
+const OntologyParser = require('./ontology-parser');
+const jsonld = require('jsonld');
+const odrl = require('./ontology/odrl.json');
+const op = require('./ontology/openpermissions.json');
+const opex = require('./ontology/opex.json');
+const geo = require('./ontology/geonames_countries.json');
+
+class OfferRenderer {
+  constructor(offerData, tag='offer') {
+    this.version = '__VERSION__';
+
+    const ontology = new OntologyParser();
+    let odrl_promise = ontology.loadOntology(odrl);
```

Comment:
I think it would be better if the `Ontology` class had methods to load these (since the files are static the `Ontology` class can know about them instead of passing them in). You could also have a `loadAll` method, e.g.:

`Promise.all([offer.loadOffer(offerData), ontology.loadAll()]).then(...)`